ovisbot/extensions/ctf/ctf.py

```diff
@@ -518,6 +518,28 @@ async def attempt(self, ctx, challname):
 
             ctf.save()
             await ctx.channel.send(f"Άμα είσαι κουνόσσιηλλος...")
+        elif chall_name == '--unsolved':
+            author = ctx.message.author.name
+            # fetch all challenges that have an empty "solved_at" attribute
+            for challenge in filter(lambda x: not x.solved_at, ctf.challenges):
+                if author in challenge.attempted_by:
+                    continue
+                challenge.attempted_by.append(author)
+                chall_channel = discord.utils.get(ctx.channel.category.channels, name=challenge.name)
+                await chall_channel.set_permissions(author, read_messages=True)
+            ctf.save()
+            await ctx.channel.send("Άτε ρε παιχτουρα μου δωκε μεσα...")
+        elif chall_name.startswith("--") and chall_name[2:] in ['web', 'pwn', 'misc', 'crypto', 'forensics', 'hardware']:
+            author = ctx.message.author.name
+            # fetch unsolved challenges which are in the mentioned category
```

Comment:
**issue (clarity):** It's not very obvious that this would only add a player to "unsolved" challenges of the corresponding category. 

**suggestion**: We can keep this but adjust the logic so that it adds players to the corresponding category (solved or not) and perhaps add a sub param of `--unsolved`. e.g. `--unsolved-web`, `--unsolved-crypto` and change the conditional to be `chall_name.startswith("--unsolved")`